### PR TITLE
Add display activated UI; various fixes

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Add display activated UI; various fixes

Add display activated panel and link for screenshot

Fix display list spacing
Dont edit schedule if not cp

Update login error messages

[beta]

## Motivation and Context
Display Settings UI improvements

## How Has This Been Tested?
Tested changes locally; updated tests as needed

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc fyi

File: web/partials/displays/display-fields.html [hunk 1]
```diff
@@ -454,7 +464,8 @@
   <div class="preview-container" ng-show="selectedSchedule">
     <div class="row border-bottom u_padding-20-vertical mx-0 mb-4">
       <div class="col-sm-12 px-0 form-inline">
-        <preview-selector ng-model="selectedSchedule" additional-class="mr-auto mb-0"></preview-selector>
+        <label class="mr-auto" ng-show="!userState.hasRole('cp')">Schedule: {{display.scheduleName}}</label>
+        <preview-selector ng-model="selectedSchedule" additional-class="mr-auto mb-0" require-role="cp"></preview-selector>
```
Comment (ezequielc) [inline web/partials/displays/display-fields.html:468]: @alex-deaconu In my test the require-role didn't hide the selector. Maybe it doesn't work in a directive. Thanks
Screen Shot 2020-08-13 at 12 57 18 PM

File: web/partials/displays/display-fields.html [hunk 2]
```diff
@@ -47,6 +47,19 @@
   <strong>{{playerActionsFactory.controlsError}}</strong>
 </div> 
 
+<div class="border-container text-center u_margin-md-top" ng-show="!displayService.statusLoading && (display | status) === 'notinstalled'">
```
Comment (ezequielc) [inline web/partials/displays/display-fields.html:50]: @alex-deaconu The Not Activated flow was changed as per https://docs.google.com/document/d/1R1l7z3_PgigETt3Fv8lNZoZhw1QSBZut_nZpEaJZQgc/edit. I think we can go with what you have here for this release, but we will update it before closing the epic. Thanks

File: web/partials/displays/display-fields.html (hunk 1 above)
Comment (alex-deaconu) [reply web/partials/displays/display-fields.html]: @ezequielc oops, totally missed that in my tests. Updated, thanks!

File: web/partials/displays/display-fields.html (hunk 2 above)
Comment (alex-deaconu) [reply web/partials/displays/display-fields.html:50]: @ezequielc missed that will update in a subsequent PR as per your suggestion. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc this is now restaged on Beta. Thanks!

**Pull request merged**